test/index.js

```diff
@@ -2106,6 +2106,7 @@ let AllTests = ContextureClient => {
     )
     let node = Tree.getNode(['root', 'pivot'])
 
+    // TODO test the loaded field population
```

Comment:
Do we still need to do this in this PR?

Reply:
Yeah I will do more work toward it separately